multiworld/envs/mujoco/sawyer_xyz/sawyer_push_nips.py

```diff
@@ -374,7 +374,12 @@ def init_angles(self):
     def log_diagnostics(self, paths, logger=None, prefix=""):
```

Comment:
why do we have log diagnostics and get diagnostics functions?

Reply:
Log: backwards compatiblility
Get: forwards compatibility

Admittedly, we can probably get rid of log, since I think even the current core (v0.1) uses get_diag

Reply:
I see, yeah it might make more sense to get rid of it then? up to you I suppose

Reply:
I removed `log_diagnostics`. Lmk if this PR looks good.